tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java

```diff
@@ -93,21 +94,13 @@ public void storageInit() throws IOException {
     hbaseConf.set(HConstants.ZOOKEEPER_CLIENT_PORT, splits[1]);
   }
 
-  @Override
-  public void setConfig(String name, String value) {
-  }
-
-  @Override
-  public void setConfigs(Map<String, String> configs) {
-  }
-
   public Configuration getHbaseConf() {
     return hbaseConf;
   }
 
   @Override
-  public long getTableVolume(URI uri) throws IOException {
-    return 0;
+  public long getTableVolume(URI uri) throws UnsupportedException {
+    throw new UnsupportedException();
```

Comment:
NotImplemented and UnsupportedException seems to be similar but they are different. NotImplemented means that we will implement it later. UnsupportedException means that this feature is not available. 

With Hbase, there is no way to make a guess about table volume. So, we should keep UnsupportedException here.